### PR TITLE
[WIP] dev/accessibility#10 : Make datepicker accessible

### PR DESCRIPTION
**Pull request by monishdeb**
Overview
----------------------------------------
A user should be able to use datepicker with the help of control keys, not just cursor actions, pretty much similar to this example
https://dequeuniversity.com/library/aria/date-pickers/sf-date-picker

Before
----------------------------------------
Datepicker widget is not accessible

After
----------------------------------------
Datepicker widget is accessible via control keys:
test-multiple-after

### CODE REVIEW

**Comment by monishdeb** [comment]
@colemanw I have moved the datepicker widget code in a separate file and added the handlers for control keys [here](https://github.com/civicrm/civicrm-core/pull/12580/files#diff-f2f7b242e17a3669f315e364800fdff0R79) 

Please have a look.

File: js/crm.datepicker.js [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+    });
+  };
+
+  function copyAttributes($source, $target, attributes) {
+    CRM._.each(attributes, function(name) {
+      if ($source.attr(name) !== undefined) {
+        $target.attr(name, $source.attr(name));
+      }
+    });
+  }
```
Comment (colemanw) [inline js/crm.datepicker.js]: If you change this in Common.js to `CRM.utils.copyAttribues` then you don't need to dupilcate the fn.

**Comment by colemanw** [comment]
@monishdeb it would be a lot easier to review this PR if it were split into two commits:

1. Move datepicker code from Common.js -> new file (with no code changes).
2. Code changes to make it accessible.

Then I could click on the 2nd commit and see what you've changed in the datepicker code.

**Comment by eileenmcnaughton** [comment]
@monishdeb test fail relates

**Comment by eileenmcnaughton** [comment]
test this please

**Comment by monishdeb** [comment]
Thanks @Clairepickle for your feedback. I will work further in this PR to mimic the accessible datepicker presented in https://dequeuniversity.com/library/aria/date-pickers/sf-date-picker Feel free to suggest any other accessible options which is not present in the example. I am marking this PR with WIP

**Comment by seamuslee001** [comment]
@monishdeb @colemanw where are we at with this?

**Pull request closed without merging**